<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1467806081816" stop="1467806085639">
	<name>Test user registration page</name>
	<title>Test user registration page</title>
	<test-cases>
		<test-case start="1467806081817" stop="1467806083194" status="failed">
			<name>Success creating</name>
			<title>Success creating</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>Expected 4 to be 5.</message>
				<stack-trace>Error: Failed expectation
    at /Users/aleksandrmedvedev/Desktop/e2e/test/protractor/specs/userForm.spec.js:30:38
    at Promise.invokeCallback_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:1329:14)
    at TaskQueue.execute_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2790:14)
    at TaskQueue.executeNext_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2773:21)
    at asyncRun (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2652:27)
    at /Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1467806083198" stop="1467806084270" status="passed">
			<name>Success not creating</name>
			<title>Success not creating</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
			</attachments>
		</test-case>
		<test-case start="1467806084271" stop="1467806084989" status="failed">
			<name>Success not delete</name>
			<title>Success not delete</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>Expected 4 to be 5.</message>
				<stack-trace>Error: Failed expectation
    at /Users/aleksandrmedvedev/Desktop/e2e/test/protractor/specs/userForm.spec.js:51:38
    at Promise.invokeCallback_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:1329:14)
    at TaskQueue.execute_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2790:14)
    at TaskQueue.executeNext_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2773:21)
    at asyncRun (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2652:27)
    at /Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1467806084990" stop="1467806085638" status="failed">
			<name>Success delete</name>
			<title>Success delete</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>Expected 4 to be 3.</message>
				<stack-trace>Error: Failed expectation
    at /Users/aleksandrmedvedev/Desktop/e2e/test/protractor/specs/userForm.spec.js:65:38
    at Promise.invokeCallback_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:1329:14)
    at TaskQueue.execute_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2790:14)
    at TaskQueue.executeNext_ (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2773:21)
    at asyncRun (/Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:2652:27)
    at /Users/aleksandrmedvedev/node_modules/selenium-webdriver/lib/promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>